### PR TITLE
Bullet Stack Fixes + Other

### PR DESCRIPTION
**Pull request by generalthrax**
## About The Pull Request

- Fixes .299 Eoehoma being 5 per stack instead of 15
- Fixes 5.56x42 CLIP being 5 per stack instead of 15
- Rebalanced the tinyass .22lr bullet stacks to be 25 instead of 15, for 100 rounds per box
- Changed the ammo can to be bulky and lets it hold 4 ammo boxes, fixes pre-spawned ammo cans to actually fit ammo boxes
- Ammo cans can only hold ammo and explosives
- Fixed a few icons and descriptions for ammo boxes
- Fixed 12g beanbag to actually have beanbag inside / along with 12g rubbershot
- Increased the speed of reloading until we get loading benches
- .22 is out of the black market since its in cargo

## Why It's Good For The Game

Fixes a bunch of stuff leftover from Bulletstacks. Rebalanced a few ammostacks to actually be usable with their guns. You will no longer have to stand in place for 3 minutes to reload a single Pounder panmag. Bug fixes good.

## Changelog

:cl:
balance: .22lr is now 25 per handful instead of 12
balance: Reloading speed from stacks doubled
balance: Increase ammo can to bulky and lets it hold 4 normal-sized ammoboxes, can only hold ammo and explosives.
balance: Adjusted price of .299 Eoehoma slightly, adds a second box to the cargo order since its 60 instead of 120
fix: Fixes a bunch of bullet stacks to have correct numbers
fix: Larger handfuls (.22lr)
fix: Mapped in ammo cans now hold their boxes
fix: 12g beanbag now has beanbag instead of rubber + beanbag
fix: Ammo box icons and descriptions
del: Removed .22lr from the blackmarket. Its in cargo already.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/ammo_can.dm [hunk 1]
```diff
@@ -1,51 +1,69 @@
...
+	STR.set_holdable(list(
+		/obj/item/storage/box/ammo,
+		/obj/item/mine,
+		/obj/item/grenade,
+		/obj/item/ammo_casing/caseless/rocket,
+		/obj/item/ammo_box/magazine/ammo_stack,
+		/obj/item/ammo_casing,
+		/obj/item/mine,
+		/obj/item/grenade
+		))
```
Comment (FalloutFalcon) [inline code/game/objects/items/storage/ammo_can.dm:27]: I would go ahead and make this a var because sub types always end up needed edge cases for this.

**Comment by rye-rice** [comment]
needs another TM?

**Comment by generalthrax** [comment]
> needs another TM?

Should be ready? Question mark?

File: code/game/objects/items/storage/ammo_can.dm [hunk 2]
```diff
@@ -7,7 +7,23 @@
 	drop_sound = 'sound/items/handling/ammobox_drop.ogg'
 	pickup_sound =  'sound/items/handling/ammobox_pickup.ogg'
 	material_flags = NONE
-	has_latches = FALSE
```
Comment (thgvr) [inline code/game/objects/items/storage/ammo_can.dm:10]: Should probably keep this false

File: code/modules/projectiles/boxes_magazines/ammo_stacks/prefab_stacks/premade_pistol_stacks.dm [hunk 3]
```diff
@@ -490,7 +492,7 @@
 /obj/item/storage/box/ammo/a44roum_hp
 	name = "box of HP .44 roumain ammo"
 	desc = "A box of .44 roumain rubbershot ammo, designed to disable targets without causing serious damage."
-	icon_state = "a44roum-rubber"
+	icon_state = "a44roum-hp"
```
Comment (thgvr) [inline code/modules/projectiles/boxes_magazines/ammo_stacks/prefab_stacks/premade_pistol_stacks.dm:495]: that doesn't seem right. is there no sprite for `-rubber` here?

**Review by thgvr — COMMENTED** [review]
looks good, just a few worries

File: code/modules/projectiles/boxes_magazines/ammo_stacks/prefab_stacks/premade_pistol_stacks.dm (hunk 3 above)
Comment (generalthrax) [reply code/modules/projectiles/boxes_magazines/ammo_stacks/prefab_stacks/premade_pistol_stacks.dm:495]: This isn't rubber. This is hollow point.

File: code/game/objects/items/storage/ammo_can.dm (hunk 2 above)
Comment (generalthrax) [reply code/game/objects/items/storage/ammo_can.dm:10]: i forgor

File: code/modules/projectiles/boxes_magazines/ammo_stacks/prefab_stacks/premade_pistol_stacks.dm (hunk 3 above)
Comment (thgvr) [reply code/modules/projectiles/boxes_magazines/ammo_stacks/prefab_stacks/premade_pistol_stacks.dm:495]: ooprs

**Pull request merged**